static/sass/_pattern_cve.scss

```diff
@@ -12,22 +12,44 @@
     color: $color-x-light;
   }
 
+  .cve-table thead tr {
+    border-bottom: 0;
+  }
+
+  .cve-table tbody tr {
+    border-top: 0;
+  }
+
   .cve-table td {
+    border-top: 1px solid $color-mid-light;
     position: relative;
   }
 
+  .cve-table .cve-table-cell {
+    border-top: 0;
+    overflow: visible;
+    padding-left: 0.75rem;
+  }
+
+  .cve-table-cell--muted {
+    @extend .cve-table-cell;
+
```

Comment:
Is this space just for readability?

Reply:
@carkod it's done automatically by stylelint - the idea is for readability